### PR TITLE
Multi tenancy

### PR DESCRIPTION
**Pull request by chanagershom**

### CODE REVIEW

File: content/composer/overview.md [hunk 1]
```diff
@@ -8,23 +8,25 @@ weight: 200
...
-Among its draggable components you'll find platform & network items (such as 'compute' node, 'database', 'web server' etc..) and you'll be able to add your own custom node types components as well as custom plugins and new interfaces. <br />
+Among its draggable components are platform and network items including 'compute' node, 'database', and 'web server' items, and so on. You can also add your own custom node type components, as well as custom plugins and new interfaces. <br />
 
-The generated result will be a downloadable tgz file containing:  <br />
+The generated result is a downloadable `tgz` file containing:  <br />
 
-- **blueprint.yaml** file which provides a TOSCA based description for the application topology and its lifecycle management. <br />
-- **Custom types and plugins** (if added by the user) and other resources the blueprint.yaml depends on.
+- The `blueprint.yaml` file, which provides a TOSCA-based description for the application topology and its lifecycle management. <br />
+- Optionally, *Custom types* and *plugins* and other resources that the `blueprint.yaml` depends on.
```
Comment (EarthmanT) [inline content/composer/overview.md]: "depends on" is probably not the right choice of words. It's more correct to say that the blueprint contains operation mappings and resource references to all kinds of scripts and config templates.

File: content/composer/overview.md [hunk 2]
```diff
@@ -45,18 +47,18 @@ To add another field, click the '+' button on the right.
...
 Inline types derive from existing types, and can define additional properties and interfaces to those available with the parent type. 
 For each node you can define interface operations and their implementation. We provide a list of available plugins for you.
 
      - **Plugins**
-Plugins can be added in the relevant tab using either a url (which points to a plugin.yaml or plugin.zip file) or using the plugin implementation file itself (uploading a local plugin.yaml or plugin.zip file).
+Plugins can be added in the relevant tab using either a URL (which points to a plugin.yaml or plugin.zip file) or using the plugin implementation file itself (uploading a local plugin.yaml or plugin.zip file).
 
      - **Relationships**
-Relationships, like types, derive from existing relationships and can add properties and interfaces. Interfaces would be defined per source and target.
+Relationships, like types, derive from existing relationships and can add properties and interfaces. Interfaces are defined per source and target.
```
Comment (EarthmanT) [inline content/composer/overview.md]: node_types ?

File: content/manager/implement-multi-tenancy.md [hunk 3]
```diff
@@ -0,0 +1,66 @@
...
+
+In the following diagram note that there are three tenants in Cloudify Manager. Tenants can be assigned groups of users, individual users, or a combination of both groups and individuals.
+
+{*Insert graphic here.*}
+
+###Integrating Multi-Tenancy with LDAP/AD###
+To connect Cloudify Manager with LDAP/AD, you must know the the URL of the service and have sufficient credentials to perform searches and so on. 
+
+####Connecting to the LDAP/AD Service####
+{*Need to add specfic process for what the user needs to do to connect. Omer, please send me the specific command that must be run on a new manager to connect it to an LDAP service.*}
```
Comment (EarthmanT) [inline content/manager/implement-multi-tenancy.md]: just checking that this does not get merged.

File: content/manager/implement-multi-tenancy.md [hunk 4]
```diff
@@ -0,0 +1,66 @@
...
+
+####How Cloudify Manager Works with the LDAP/AD Service####
+When a user logs in to Cloudify Manager, their credentials are passed to the LDAP/AD service for authentication. By default, all users in the LDAP/AD service are authenticated to Cloudify Manager, however only users who have specific permissions for a tenant can access it. 
+
+When a user logs into Cloudify Manager, the service authenticates the user and returns a list of any groups to which the user belongs. If a Cloudify Manager instance has specified a group name for a tenant that has the same name as a group specified in LDAP/AD, all users in that group can access the tenant (unless an individual user is specifically suspended.) In addition, individual users can be given access to a tenant. The following graphic indicates how Cloudify Manager interacts with an LDAP/AD service. 
+
+{*Insert graphic here.*}
+
+####Managing Users in the Multi-Tenancy Environment####
+There are a number of actions related to user access to tenants that an **`administrator`** user can perform.
```
Comment (EarthmanT) [inline content/manager/implement-multi-tenancy.md]: what are they

File: content/manager/implement-multi-tenancy.md (hunk 3 above)
Comment (chanagershom) [inline content/manager/implement-multi-tenancy.md]: This is still in draft format and won't get merged in the next few days (or before this is fixed.) Thanks!

**Pull request closed without merging**